lib/js-api/src/api/content-rest-api/model/ruleRef.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface RuleParameter {
```

Comment:
this should not be part of the JS-API library as it has nothing to do with the REST api calls and ACS backend

Reply:
This interface is used by the RuleRef model, which is in turn being used in the newly created REST api call for saving the extension state on the ACS backend.